### ISSUE TITLE
Error when attempting to update existing CNAME record using zoneHostId

### ISSUE THREAD

**Issue by anthcourtney** [widdix/node-route53-updater#3]
Hey guys,

I've successfully used route53-updater to create a CNAME for a given hostedZoneId, however when the same command is run a subsequent time (as part of the userdata in an ASG), I'm hitting an error because the CNAME already exists.

I was of the understanding that the UPDATE action should work for new and existing records.

Please find the output of the command below.

(Note I have sanitised the hostedZoneId and recordSetName values.)

```
$ route53-updater --action UPDATE --hostedZoneId ZXXXXXXXXX --recordSetName my.domain.name.

route53-updater@1.0.0 /usr/lib/node_modules/route53-updater
├── assert-plus@1.0.0
├── minimist@1.2.0
├── underscore@1.8.3
├── async@1.5.2
└── aws-sdk@2.4.1 (xml2js@0.4.15, sax@1.1.5, jmespath@0.15.0, xmlbuilder@2.6.2)
+ route53-updater --action UPDATE --hostedZoneId ZXXXXXXXXXX --recordSetName my.domain.name
Tried to create resource record set [name='my.domain.name.', type='CNAME'] but it already exists InvalidChangeBatch: Tried to create resource record set [name='my.domain.name.', type='CNAME'] but it already exists
    at Request.extractError (/usr/lib/node_modules/route53-updater/node_modules/aws-sdk/lib/protocol/rest_xml.js:44:29)
    at Request.callListeners (/usr/lib/node_modules/route53-updater/node_modules/aws-sdk/lib/sequential_executor.js:105:20)
    at Request.emit (/usr/lib/node_modules/route53-updater/node_modules/aws-sdk/lib/sequential_executor.js:77:10)
    at Request.emit (/usr/lib/node_modules/route53-updater/node_modules/aws-sdk/lib/request.js:615:14)
    at Request.transition (/usr/lib/node_modules/route53-updater/node_modules/aws-sdk/lib/request.js:22:10)
    at AcceptorStateMachine.runTo (/usr/lib/node_modules/route53-updater/node_modules/aws-sdk/lib/state_machine.js:14:12)
    at /usr/lib/node_modules/route53-updater/node_modules/aws-sdk/lib/state_machine.js:26:10
    at Request.<anonymous> (/usr/lib/node_modules/route53-updater/node_modules/aws-sdk/lib/request.js:38:9)
    at Request.<anonymous> (/usr/lib/node_modules/route53-updater/node_modules/aws-sdk/lib/request.js:617:12)
    at Request.callListeners (/usr/lib/node_modules/route53-updater/node_modules/aws-sdk/lib/sequential_executor.js:115:18)
```

**Comment by andreaswittig** [comment]
Hi @anthcourtney, thanks for your feedback.

Are you starting multiple EC2 instances with the help of Auto Scaling? And all of them are updating the same Route53 record set?

**Comment by anthcourtney** [comment]
Hello,

My ASG is 1:1:1, so there is only one instance running at any one time. 

I figured that the update support would cater for when an instance terminates, the replacement instance (spun up by the ASG launch configuration) could then update the record.

cheers,
Anth

**Comment by andreaswittig** [comment]
Unfortunately I'm still not able to reproduce your issue. Could you try to describe your environment and process a little bit deeper, please?

Nevertheless, I have created a PR with a change that might fix your issue. Waiting for @michaelwittig to review.

**Comment by andreaswittig** [comment]
I've published version 1.0.1 including this improvement.
